tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Use the same tsconfig settings as the other branches

Things like target are configured to be different --> es6 vs es5
Recommend just copying over the config and adding to it if required
Setting the branch to be downstream from `update/typescript` may help